src/lib/Components/Consignments/Screens/Artist.tsx

```diff
@@ -66,11 +66,19 @@ export default class Artist extends React.Component<Props, State> {
   }
 
   render() {
+    const isPad = Dimensions.get("window").width > 700
```

Comment:
We should really factor this logic out into a helper function sometime.

Reply:
Yeah, there's the onLayout technique, but it's a lot of code and doubles the render pass :-/